app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <integer name="duration_anim_300_ms">300</integer>
```

Comment:
Please make this contextualized or inline it. The name '300ms' is the same as the value. Instead, we should specify how it should be used in its name, e.g. "default_fade_anim_time_ms" or similar.

Reply:
Updated names to `default_fade_anim_time_ms` and `topic_play_animation_time_ms`